SharpXMPP.WebSocket/XmppWebSocketConnection.cs

```diff
@@ -113,23 +113,21 @@ public override Task ConnectAsync(CancellationToken token)
         {
             return Task.Run(async () =>
```

Comment:
_(Minor)_ I believe we don't need this `Task.Run` anymore, as the new DNS client supports asynchrony out of the box?

Reply:
It will break existing XmppConnection API and anyway it is not a subject of this PR, maybe it should addressed in a new issue/PR